### PR TITLE
Fint oneroster rest provider

### PR DESCRIPTION
**Pull request by erlingjahr**

### CODE REVIEW

File: charts/fint-oneroster-rest-provider/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: fint-oneroster-rest-provider
+description: A Helm chart for Kubernetes
```
Comment (mmedalen) [inline charts/fint-oneroster-rest-provider/Chart.yaml]: add better description

File: charts/fint-oneroster-rest-provider/values-oneroster-rest-provider-agder.yaml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+environment: beta
+onePasswordItem: 
+  name: oneroster-agder
+  create: false
+
+service:
+  type: LoadBalancer
+  port: 8080
+  
+deployment:
```
Comment (mmedalen) [inline charts/fint-oneroster-rest-provider/values-oneroster-rest-provider-agder.yaml]: is "deployment" required in this file? could you not have had the values ​​without?

File: charts/fint-oneroster-rest-provider/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "fint-oneroster-rest-provider.name" . }}
+  annotations:
+    service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```
Comment (mmedalen) [inline charts/fint-oneroster-rest-provider/templates/service.yaml]: possible use helmet on this value aswell? 
e.g
metadata:
      {{- with .Values.service.annotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}

**Review by mmedalen — COMMENTED** [review]
maybe change the value files to just the county name? e.g. value-vtfk.yaml etc. instead of having the application name in the file name? it's a bit redundant.

File: charts/fint-oneroster-rest-provider/Chart.yaml (hunk 1 above)
Comment (erlingjahr) [reply charts/fint-oneroster-rest-provider/Chart.yaml]: changed to better description

File: charts/fint-oneroster-rest-provider/values-oneroster-rest-provider-agder.yaml (hunk 2 above)
Comment (erlingjahr) [reply charts/fint-oneroster-rest-provider/values-oneroster-rest-provider-agder.yaml]: Used to split the values file into service and deployment sections for better readability

File: charts/fint-oneroster-rest-provider/templates/service.yaml (hunk 3 above)
Comment (erlingjahr) [reply charts/fint-oneroster-rest-provider/templates/service.yaml]: changed to use values instead

**Comment by erlingjahr** [comment]
> maybe change the value files to just the county name? e.g. value-vtfk.yaml etc. instead of having the application name in the file name? it's a bit redundant.

Changed files :-)

**Review by mmedalen — APPROVED** [review]

**Pull request merged**